pages/mint-nft/index.vue

```diff
@@ -3,452 +3,466 @@
+            🎉 Success!
+          </template>
 
-        <template v-if="shouldShowDownloadLink">
-          <UButton
-            label="Download nft_class.json"
-            :disabled="isLoading"
-            variant="outline"
-            @click="onDownloadClassJSON"
-          />
           <UButton
-            label="Download nft_default.json"
+            label="Download NFT result csv"
             :disabled="isLoading"
             variant="outline"
-            @click="onDownloadDefaultClassJSON"
+            @click="onDownloadCSV"
           />
-          <UButton
-            label="Download nfts.csv"
-            :disabled="isLoading"
-            variant="outline"
-            @click="onDownloadNftsCSV"
-          />
-        </template>
 
-        <template #footer>
-          <UButton
-            label="View your NFT"
-            variant="outline"
-            target="_blank"
-            :to="`${likerLandURL}/nft/class/${encodeURIComponent(classId)}`"
-          />
-          <div class="p-[4px] border-[2px] border-[#f59e0b] rounded-[0.375rem]">
+          <template v-if="shouldShowDownloadLink">
+            <UButton
+              label="Download nft_class.json"
+              :disabled="isLoading"
+              variant="outline"
+              @click="onDownloadClassJSON"
+            />
             <UButton
-              :to="{ name: 'nft-book-store-new', query: { class_id: classId, count: nftMintCount } }"
-              label="Continue to publish NFT Book / 繼續上架"
-              variant="solid"
-              color="orange"
+              label="Download nft_default.json"
+              :disabled="isLoading"
+              variant="outline"
+              @click="onDownloadDefaultClassJSON"
             />
-          </div>
-        </template>
-      </UCard>
-
-      <UProgress v-if="isLoading" animation="carousel">
-        <template #indicator>
-          Loading...
-        </template>
-      </UProgress>
-      <EditISCNMetadataModal
-        ref="editISCNRef"
-        v-model="showEditISCNModal"
-        :class-id="classId"
-        @save="onSaveISCN"
-      />
+            <UButton
+              label="Download nfts.csv"
+              :disabled="isLoading"
+              variant="outline"
+              @click="onDownloadNftsCSV"
+            />
+          </template>
+
+          <template #footer>
+            <UButton
+              label="View your NFT"
+              variant="outline"
+              target="_blank"
+              :to="`${likerLandURL}/nft/class/${encodeURIComponent(classId)}`"
+            />
+            <div class="p-[4px] border-[2px] border-[#f59e0b] rounded-[0.375rem]">
+              <UButton
+                v-if="isRestockingNFT"
+                :to="{
+                  name: 'nft-book-store-status-classId',
+                  params: { classId },
+                  query: { count: nftMintCount }
+                }"
+                label="Restock existing listing/ 新庫存上架"
```

Comment:
```suggestion
                label="Restock existing listing / 新庫存上架"
```

Reply:
[🚸 Update label formatting and reset restocking state on ISCN ID input](https://github.com/likecoin/nft-book-press/pull/330/commits/e33bae6161bd731a7b11ca6a0cf9f063e5aa8b1c)